node/script/test_dal_consumer_ws/main.go

```diff
@@ -0,0 +1,100 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"strconv"
+	"sync"
+	"time"
+
+	"bisonai.com/orakl/node/pkg/common/types"
+	"bisonai.com/orakl/node/pkg/utils/request"
+	"bisonai.com/orakl/node/pkg/wss"
+	"github.com/rs/zerolog/log"
+)
+
+type Subscription struct {
+	Method string   `json:"method"`
+	Params []string `json:"params"`
+}
+
+func main() {
+	ctx := context.Background()
+	chain := "baobab"
+	key := ""
+	configs, err := fetchConfigs()
+	if err != nil {
+		panic(err)
+	}
+	subscription := Subscription{
+		Method: "SUBSCRIBE",
+		Params: []string{},
+	}
+
+	for _, configs := range configs {
+		subscription.Params = append(subscription.Params, "submission@"+configs.Name)
+	}
+
+	wsEndpoint := fmt.Sprintf("ws://dal.%s.orakl.network/ws", chain)
+	// wsEndpoint := "ws://localhost:8090/ws"
```

Comment:
should we remove this if not used?

Reply:
I left the comment so that I can use it easily, for personal use, but if you insist, will remove 😅